.eslintrc

```diff
@@ -0,0 +1,48 @@
+{
+  "extends": [
+    "lisk-base"
+  ],
+  "plugins": [
+    "import"
+  ],
+  "globals": {
+    "ipc": true,
+    "PRODUCTION": true,
+    "TEST": true,
+    "env.mocha": true
```

Comment:
Ah sorry I wasn't clear: this goes in the `env` section, like `browser`, and `node`